src/libs/antares/writer/zip_writer.cpp

```diff
@@ -46,7 +46,7 @@ static std::unique_ptr<mz_zip_file> createInfo(const std::string& entryPath)
 }
 
 template<class ContentT>
-void ZipWriteJob<ContentT>::onExecute()
+void ZipWriteJob<ContentT>::writeEntry()
```

Comment:
I see a few `logs.error()` below. Could we also `throw` exceptions now ?

Reply:
Indeed: it will change a little the behaviour from what it is now, but I think it's better to get an error instead of having an incomplete/corrupt result without knowing it (who looks at the log? :) ).